noxfile.py

```diff
@@ -7,6 +7,12 @@ def fmt_check(s: nox.Session) -> None:
     s.run("black", "--check", ".")
 
 
+@nox.session(tags=["fmt"], venv_backend="none")
```

Comment:
Unsure if `tags=["fmt"]` is needed, presumably session would be invoked by `nox -s fmt`.